### ISSUE TITLE
attribute is repeated twice in python

### ISSUE THREAD

**Issue by zhangzq** [doxygen/doxygen#10973]
**Describe the bug**
The python code below:

```
class Test:
    def __init__(self):
        ## var a
        self.a: int = 1

        print(self.a)
```

will output two attributes with same name 'a'.

**Screenshots**

![image](https://github.com/doxygen/doxygen/assets/3370086/f8702e90-a1e0-4e9d-ada6-37243a3fde7b)

**To Reproduce**

The above small snipets with default doxyfile.

**Expected behavior**

There should only one 'a'.

**Version**

1.12.0 (4a9be1303830cb78f8def8b60888cd6e23afd34d*)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10975

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
